src/deployable_example.cpp

```diff
@@ -146,6 +147,8 @@ void debug(float message) {
   }
 }
 
+bool flash_preferences = false;
```

Comment:
move the bool to credentials.h so all the modifications are in one place ?  maybe rename to be more clear?